code/modules/cm_marines/equipment/sentries.dm

```diff
@@ -227,7 +227,8 @@
 	var/datum/effect_system/spark_spread/spark_system //The spark system, used for generating... sparks?
 	var/obj/item/cell/cell = null
 	var/burst_fire = FALSE
-	var/obj/machinery/camera/camera = null
+	var/obj/machinery/camera/camera
+	var/has_camera = TRUE
```

Comment:
Add it as a turret bitflag perhaps? It's what, the sixth boolean var added to this object?